### PR TITLE
Fix user context exp

### PR DESCRIPTION
**Pull request by christian14b**
Track experiment to only non-depositor or non-lender users

### CODE REVIEW

File: src/components/BorrowerProfile/LoanStory.vue [hunk 1]
```diff
@@ -206,7 +211,7 @@ export default {
 			this.isoCode = loan?.geocode?.country?.isoCode ?? '';
 			this.tags = loan?.tags ?? [];
 			this.gender = loan?.gender ?? '';
-			this.sector = loan?.sector ?? '';
+			this.sector = loan?.sector ?? { name: '' };
```
Comment (dyersituations) [inline src/components/BorrowerProfile/LoanStory.vue:214]: Is this change needed with the optional chaining I added?

**Review by dyersituations — APPROVED** [review]

File: src/components/BorrowerProfile/LoanStory.vue (hunk 1 above)
Comment (mcstover) [reply src/components/BorrowerProfile/LoanStory.vue:214]: I think it would overwrite the initial value which contains this stub object in `data` above if it came back undefined.

**Review by mcstover — APPROVED** [review]
